package.nls.json

```diff
@@ -420,8 +420,13 @@
     "AWS.s3.downloadFile.progressTitle": "Downloading {0}...",
     "AWS.s3.downloadFile.saveButton": "Download",
     "AWS.s3.downloadFile.error.general": "Failed to download file {0}",
+    "AWS.s3.fileViewer.message.tempCreation": "Temp folder for FileViewer created with location: {0}",
     "AWS.s3.fileViewer.warning.noSize": "File size couldn't be determined. Continue with download?",
     "AWS.s3.fileViewer.warning.4mb": "File size is more than 4MB. Continue with download?",
+    "AWS.s3.fileViewer.message.fileKey": "Retrieving and displaying file {0}",
+    "AWS.s3.fileViewer.message.fileLocation": "File to be opened is: {0}",
+    "AWS.s3.fileViewer.message.sizeLimitConfirmed": "Confirmed download, continuing",
+    "AWS.s3.fileViewer.error.download": "Error downloading: ${err.toString()}",
```

Comment:
"Error downloading file {0} from S3: {1}"

Also edit in code. Params should be:
{0}: file name
{1}: error